src/hosted-form/hosted-form.ts

```diff
@@ -46,25 +45,23 @@ export default class HostedForm {
     }
 
     async submit(payload: OrderPaymentRequestBody): Promise<void> {
-        return await this._getNumberField().submit(
+        return await this._getFirstField().submit(
             this._fields.map(field => field.getType()),
             this._payloadTransformer.transform(payload)
         );
     }
 
     async validate(): Promise<void> {
-        return await this._getNumberField().validate();
+        return await this._getFirstField().validate();
```

Comment:
why do we only validate the first field?

Reply:
`validate` actually validates the entire form. The reason is because some fields are related - i.e.: CVV depends on the card number. I could probably rename it to `validateForm`.

Reply:
yeah I think validate is meant to validate the whole form, I was mostly asking how by validating the first field, we also validate other fields, by just looking at `this._getFirstField().validate();` it's hard to guess how it validates the whole form